amd_openvx_extensions/amd_rpp/source/CropMirrorNormalize.cpp

```diff
@@ -21,127 +21,116 @@ THE SOFTWARE.
 */
 
 #include "internal_publishKernels.h"
-#define NUM_OF_DIMS 5
 
-struct CropMirrorNormalizeLocalData
-{
+struct CropMirrorNormalizeLocalData {
     RPPCommonHandle handle;
     rppHandle_t rppHandle;
-    Rpp32u device_type;
+    Rpp32u deviceType;
     Rpp32u nbatchSize;
     RppPtr_t pSrc;
     RppPtr_t pDst;
-    vx_uint32 *start_x;
-    vx_uint32 *start_y;
-    vx_uint32 *crop_h;
-    vx_uint32 *crop_w;
     vx_float32 *mean;
-    vx_float32 *std_dev;
+    vx_float32 *stdDev;
     vx_uint32 *mirror;
-    RpptDescPtr src_desc_ptr;
+    RpptDescPtr srcDescPtr;
     RpptDesc srcDesc;
     RpptDesc dstDesc;
-    RpptDescPtr dst_desc_ptr;
-    RpptROI *roi_tensor_ptr;
+    RpptDescPtr dstDescPtr;
+    void *roiTensorPtr;
+    RpptROI *roiPtr;
     RpptRoiType roiType;
-    Rpp32u layout;
-    size_t in_tensor_dims[NUM_OF_DIMS];
-    size_t out_tensor_dims[NUM_OF_DIMS];
-    vx_enum in_tensor_type;
-    vx_enum out_tensor_type;
+    Rpp32s inputLayout;
+    Rpp32s outputLayout;
+    size_t inputTensorDims[NUM_OF_DIMS];
```

Comment:
dims and type has to be part of tensor descriptor

Reply:
The tensor descriptor in RPP stores the dims in a particular structure format hence we are storing dims in the array and converting to the RPP struct format for now, Once change is made in RPP to accept dims array we will make changes accordingly in OpenVX extensions